linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -213,6 +213,14 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL_TIMER), threadpool_timer_start, THANDLE(T
 
 **SRS_THREADPOOL_LINUX_07_091: [** `threadpool_timer_start` shall initialize the lock guarding the timer state by calling `srw_lock_ll_init`. **]**
 
+**SRS_THREADPOOL_LINUX_07_096: [** threadpool_timer_start shall call `lazy_init` with `do_init` as initialization function.  **]**
```

Comment:
>`do_init` [](http://example.com/codeflow?start=86&length=9)

Consider putting the do_init signature right after this function (ln 239)
And having the specs for do_init placed there too.

/consider

Reply:
done